### DISCUSSION TITLE
v3.0.0.rc1

### DISCUSSION THREAD

**Discussion by pcai** [savonrb/savon#1011] [Announcements]
## What's Changed

* HTTPI to Faraday by @LukeIGS in https://github.com/savonrb/savon/pull/998

**Full Changelog**: https://github.com/savonrb/savon/compare/v2.15.1...v3.0.0.rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/savonrb/savon/releases/tag/v3.0.0.rc1'>v3.0.0.rc1</a>.</em>

**Comment by rubiii** [comment]
We shipped the Faraday transport planned for 3.0 as an opt-in feature in [v2.17.0](https://github.com/savonrb/savon/blob/main/CHANGELOG.md#2170---2026-05-19).
For the current plan for 2.x and 3.0 see https://github.com/savonrb/savon/discussions/1060.
